lib/src/lib.rs

```diff
@@ -202,4 +202,34 @@ pub mod serde_with {
             T::decode_bytes(bytes).map_err(serde::de::Error::custom)
         }
     }
+
+    pub struct RlpHexBytes {}
```

Comment:
`RlpBytes` bytes not working sometimes or is this for nicer serialization?

Reply:
only changed in native return, as it now returns like:
[1,
 2,
 3,
...
].
So, I add this one to return "0x010203...". just for beautify the return json. no real advantage.